### ISSUE TITLE
Can't run compiled version on real hardware

### ISSUE THREAD

**Issue by Anh302kay** [masterfeizz/Xash3DS#56]
Compiled Xash3DS using latest version of libctru and Citro3D. The SDK(idk what to call it) was compiled using a downgraded version of libctru. Works perfectly in citra, but crashes on a new 3DS XL, when loading into a map. Can load into main menu.

**Comment by quadblip** [comment]
Are you sure you have the files in the right places?
(Half-Life directory)/valve -> sd:/xash3d/
And have you deleted the .cfg files as said by the install instructions?
Also, can you send me the .3dsx file you compiled? (this is for testing reasons)

**Comment by Anh302kay** [comment]
Yes, I had everything setup correctly, as it worked perfectly fine with the precompiled versions. IIRC the CIA was the one that makes it into menu before crashing, and the 3DSX crashes on start-up.

[Xash3DS.zip](https://github.com/user-attachments/files/19160031/Xash3DS.zip)
